docs/CHANGELOG.md

```diff
@@ -17,6 +17,10 @@ title: Changelog
 
     *Josh Clayton*
 
+* Implement predicate method for polymorphic slots
```

Comment:
```suggestion
* Add predicate method support to polymorphic slots.
```

Super minor, just copying some previous changelog entries wording for consistency.